packages/fxa-settings/src/pages/Signin/SigninRecoveryChoice/container.tsx

```diff
@@ -95,7 +106,7 @@ export const SigninRecoveryChoiceContainer = (_: RouteComponentProps) => {
     } catch (err) {
       if (err.errno && AuthUiErrorNos[err.errno]) {
         if (err.errno === AuthUiErrors.INVALID_TOKEN.errno) {
-          navigateWithQuery('/signin');
+          // navigateWithQuery('/signin');
```

Comment:
Bah, with my next push I'll revert this (I'd commented some stuff out when debugging), as well as reverting that `fullScreen` removal currently on L119. (I was playing with adding AppLayout in there since the "blip" is noticeable between pages but then decided to not change that here.)